extension/src/counter_agg.rs

```diff
@@ -92,7 +92,8 @@ impl<'input> CounterSummary<'input> {
         prev: Option<CounterSummary>,
         next: Option<CounterSummary>,
     ) -> CounterSummary<'static> {
-        let prev = prev.map(|summary| {
+        let prev = if self.first.ts > interval_start {
```

Comment:
The test still passes if this is `>=` - can you add a test that would fail in that case?

Reply:
Indeed!  I don't know what confused me, but thanks.